compose.selenium.yml

```diff
@@ -1,4 +1,4 @@
-version: '3.7'
+version: "3.7"
```

Comment:
I think version is deprecated and can be safely removed.

Reply:
I will update it in separate PR later today or tomorrow morning